midend/interpreter.cpp

```diff
@@ -54,6 +56,7 @@ SymbolicValue* SymbolicValueFactory::create(const IR::Type* type, bool uninitial
 
 bool SymbolicValueFactory::isFixedWidth(const IR::Type* type) const {
     type = typeMap->getType(type, true);
+    type = type->to<IR::Type_Type>()->type;
```

Comment:
these two lines can be replaced with `typeMap->getTypeType(type);`

Reply:
Thanks for the advice, I`ll do it